core/src/test/java/greencity/validator/ImageValidatorTest.java

```diff
@@ -0,0 +1,62 @@
+package greencity.validator;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Stream;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.MethodSource;
+import org.junit.jupiter.params.provider.NullSource;
+import org.springframework.mock.web.MockMultipartFile;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+import org.springframework.web.multipart.MultipartFile;
+
+@ExtendWith(SpringExtension.class)
+class ImageValidatorTest {
+
+    private ImageValidator imageValidator;
+
+    private static final List<String> validImageContentTypes = Arrays.asList("image/jpeg", "image/png", "image/jpg");
+    private static final List<String> invalidContentTypes = Arrays.asList("image/bmp", "image/gif", "audio/mp3",
+            "video/mp4", "text/plain", "application/json");
```

Comment:
Static and final fields have to be before non-static and non-final

Reply:
moved validImageContentTypes and invalidImageContentTypes to MethodSource methods